content/kapacitor/v1.6/about_the_project/releasenotes-changelog.md

```diff
@@ -7,6 +7,20 @@ menu:
     name: Release notes
 ---
 
+## v1.6.3 [2022-01-20]
+
+### Features
+
+- Support for the `attributes` field in Alerta node.
+- Add BigPanda handler options.
+- The `env var` config can now handle new items.
```

Comment:
Does "handle" mean accept or add in this context? Do "items" refer to new env configurations?

Reply:
Changed wording